stock_storage_type/models/stock_location.py

```diff
@@ -357,18 +357,6 @@ def _get_pack_putaway_strategy(self, putaway_location, quant, product):
 
         for package_sequence in package_locations:
             pref_loc = package_sequence.location_id
-            if (
-                pref_loc.pack_putaway_strategy == "none"
-                and pref_loc.select_allowed_locations(
-                    package_storage_type, quant, product
-                )
-            ):
-                _logger.debug(
-                    "No putaway strategy defined on location %s and package "
-                    "storage type %s allowed."
-                    % (pref_loc.complete_name, package_storage_type.name)
-                )
-                return pref_loc
```

Comment:
After reviewing this part of code with Laurent, we could not find any use as this is the same as what is happening after the if